coral-schema/src/main/java/com/linkedin/coral/schema/avro/SchemaUtilities.java

```diff
@@ -211,12 +213,11 @@ static void appendField(@Nonnull String fieldName, @Nonnull Schema.Field field,
     Preconditions.checkNotNull(field);
     Preconditions.checkNotNull(fieldAssembler);
 
-    JsonNode defaultValue = field.defaultValue();
-
     SchemaBuilder.GenericDefault genericDefault = fieldAssembler.name(fieldName).type(field.schema());
-    if (defaultValue != null) {
+    try {
+      Object defaultValue = AvroCompatibilityHelper.getGenericDefaultValue(field);
```

Comment:
Is it because `field.defaultValue()` is no longer supported in new avro versions?
If so, there are some other `field.defaultValue()` in this file, should we also change them?

Reply:
good point.